src/org/usfirst/frc/team1922/robot/commands/DriveStraight_Command.java

```diff
@@ -13,10 +13,10 @@
 	//Unused, but unsure if I should comment out 
 	private double destination, start, heading;
 	
-	public DriveStraight_Command(int feet) {
+	public DriveStraight_Command(double d) {
```

Comment:
Changing this to a double was a a good idea but `feet` was a better name. `d` gives a reader of the code no indication of what the number is for. It's pretty easy to guess what the command does based on the name, but with `d` you can't tell whether it's in feet, inches, meters, or anything else.